changelog/1316.doc.rst

```diff
@@ -1,2 +1,2 @@
 Added a discussion of test parametrization with argument unpacking to
-the testing guide in the contributor guide.
+the testing guide in the |contributor guide|.
```

Comment:
This is currently not render correctly on RTD.

![image](https://user-images.githubusercontent.com/29869348/155810132-a3d21905-ec3f-4d36-b3a1-29846e21da06.png)